canonical/canonical.com
# feat: /data/kafka/what-is-kafka build

## Done

- Built page as per [doc](https://docs.google.com/document/d/1IThZcfKZcv-ArqU_0n8A181AGkQPUn_Iq1O4sqk5GTs/edit?tab=t.0) and [design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=2135-30990&node-type=instance&m=dev) 

## QA

- View the site locally in your web browser at: https://canonical-com-1408.demos.haus/data/kafka/what-is-kafka
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-15054
